interfaces/seccomp/template.go

```diff
@@ -500,3 +500,10 @@ pwritev
 # of socket(), bind(), connect(), etc individually.
 socketcall
 `)
+
+// defaultHookTemplate contains default seccomp template for hooks.
+// It can be overridden for testing using MockTemplate().
```

Comment:
Please add rationale for this and reference the golang bug and the seccomp bug report